### DISCUSSION TITLE
On search results pages should we hide no indexed pages

### DISCUSSION THREAD

**Discussion by AmnestyAM** [amnestywebsite/humanity-theme#75] [Enhancement Requests]
![image](https://user-images.githubusercontent.com/37809410/212338370-6b0acac8-e0ab-4011-86a7-2db2614fa5a9.png)

**Comment by jaymcp** [comment]
it may be better to implement a feature similar to the "hide from curation blocks" one, if we're going to need to hide pages, for performance reasons. noindex uses meta, which is slow to query.
